cmake/cmake_celix/BundlePackaging.cmake

```diff
@@ -107,7 +107,7 @@ Optional arguments are:
 - GROUP: The group the bundle is part of. This will be used as Bundle-Group manifest entry. Default this is empty (no group).
 - VERSION: The bundle version. This will be used for the Bundle-Version manifest entry. In combination with SOURCES the version will also be used to set the activator library target property VERSION and SOVERSION.
   For SOVERSION only the major part is used. Expected scheme is "<major>.<minor>.<path>". Default version is "0.0.0"
-- FILENAME: The filename of the bundle file. Default is <bundle_target_name>.zip.
+- FILENAME: The filename of the bundle file, without extension. Default is <bundle_target_name>.zip.
```

Comment:
`Default is <bundle_target_name>.zip.`, but the .zip part is removed now, right?